jbpm-console-ng-showcase/pom.xml

```diff
@@ -691,6 +697,21 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- Needed for the Deployment Descriptor Editor -->
```

Comment:
Really!!? I'd have thought it might have need the XML Text Editor in kie-wb-common.. but the DRL editor!?!